id-repository/id-repository-identity-service/src/main/java/io/mosip/idrepository/identity/validator/IdRequestValidator.java

```diff
@@ -323,6 +317,27 @@ private void validateVerifiedAttributes(Map<String, Object> requestMap)
 			String idSchema = restTemplate.getForObject(verifiedAttributesSchemaUrl, String.class);
```

Comment:
Can we cache this schema instead of fetching this on every request?

Reply:
taken care